src/main/drivers/accgyro_spi_mpu6500.c

```diff
@@ -163,8 +167,32 @@ static void mpu6500AccRead(int16_t *accData)
     accData[Z] = (int16_t)((buf[4] << 8) | buf[5]);
 }
 
+#ifdef COLIBRI_RACE
+static void mpu6500SpiInit(void)
```

Comment:
But where i should put this into? because i saw the other MPU driver is doing the same thing, that's why i put it there. Could you please show me where i should put this code at?